src/main/resources/README.txt

```diff
@@ -5,5 +5,55 @@
 |__/|__/_/_/ /_/\__,_/\__,_/ .___/ 
                           /_/      
 
-Windup ${project.version} is a tool used to simplify Java application migrations.
+Windup ${project.version} is an extensible and customizable rule-based tool that helps simplify migration of Java applications.
 
+Windup examines application artifacts, including project source directories and applications archives, 
+then produces an HTML report highlighting areas that need changes. 
+
+Windup can be used to migrate Java applications from previous versions of Red Hat JBoss Enterprise Application Platform 
+or from other containers, such as Oracle WebLogic Server or IBM® WebSphere® Application Server.
```

Comment:
good catch, again copy and paste error, I think i will remove the (R) to not complicate it.